src/components/my-profile/ReservedRockets/ReservedRockets.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+const ReservedRockets = () => {
+  const reservedRockets = useSelector((state) => state.rockets
+    .filter((rocket) => rocket.reserved));
+
+  return (
+    <div className="flex-1">
+      <h2 className="text-3xl">My Rockets</h2>
+      <ul className="mt-3 border-2 w-4/12 rounded">
```

Comment:
let's add a width of 50% so that the text will have full width.
```suggestion
      <ul className="mt-3 border-2 w-1/2 rounded">
```
![image](https://user-images.githubusercontent.com/106348671/193078096-f3a0bba0-21ec-4b14-a075-0cb8df0cc321.png)

Reply:
I will add these changes in my next PR. Thanks :)